plugins/modules/rds_cluster.py

```diff
@@ -463,6 +470,42 @@
     engine: aurora-postgresql
     state: present
     db_instance_class: 'db.t3.medium'
+
+- name: Remove a cluster from global DB (do not delete)
+  amazon.aws.rds_cluster:
+    db_cluster_identifier: '{{ cluster_id }}'
+    global_cluster_identifier: '{{ global_cluster_id }}'
+    remove_from_global_db: true
+
+- name: Remove a cluster from global DB and Delete without creating a final snapshot
+  amazon.aws.rds_cluster:
+    engine: aurora
+    password: "{{ password }}"
+    username: "{{ username }}"
+    cluster_id: "{{ cluster_id }}"
+    skip_final_snapshot: true
+    remove_from_global_db: true
+    wait: true
+    state: absent
```

Comment:
This is resulting in the following error - `botocore.exceptions.WaiterError: Waiter cluster_available failed: Max attempts exceeded. Previously accepted state: Matched expected service error code: DBClusterNotFoundFault`

The task is 
`- name: Delete replica cluster
      amazon.aws.rds_cluster:
        db_cluster_identifier: "{{ test_replica_cluster_name }}"
        global_cluster_identifier: "{{ test_global_cluster_name }}"
        engine: "{{ rds_engine }}"
        engine_version: "{{ rds_engine_version }}"
        skip_final_snapshot: true
        remove_from_global_db: true
        region: "{{ test_replica_cluster_region }}"
        state: absent`
        
      Shouldn't this task work?

Reply:
it should work, can you retry running the tests? I've seen aws being flakey at times on this
I'll try running it again as well from my end and keep you posted

Reply:
I've noticed that the first attempt at running it always fails, but it seems to pass when run a second time. The issue is consistent, so it's better to address it and find a solution to prevent this from happening.

Reply:
i'm not sure what can be done to address aws side errors, maybe increasing the retries and wait time?
in waiter here https://github.com/ansible-collections/amazon.aws/pull/1705/files#diff-9eb5385261889b20f465ea56fd4fe899498b77edc95f084f8b14c2075052c3d7R550-R554

Reply:
@GomathiselviS increase wait time fro. 60 seconds to 300, should work now

Reply:
worked fine, tests passed in first run for me

```
TASK [rds_cluster_modify : Assert that primary cluster is NOT a part of global db] ***
ok: [testhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [rds_cluster_modify : Delete replica cluster] *****************************
ok: [testhost] => {"changed": false, "resource_actions": ["rds:DescribeDBClusters"]}

TASK [rds_cluster_modify : Delete instance connected to primary cluster] *******
changed: [testhost] => {"changed": true, "resource_actions": ["rds:ListTagsForResource", "rds:DeleteDBInstance", "rds:DescribeDBInstances"]}

TASK [rds_cluster_modify : Delete primary cluster] *****************************
changed: [testhost] => {"changed": true, "resource_actions": ["rds:DeleteDBCluster", "rds:DescribeDBClusters"]}

TASK [rds_cluster_modify : Delete global db] ***********************************
changed: [testhost] => {"changed": true, "resource_actions": ["cloudcontrolapi:GetResource", "cloudcontrolapi:ListResourceRequests", "cloudcontrolapi:DeleteResource"], "result": {}}

PLAY RECAP *********************************************************************
testhost                   : ok=31   changed=14   unreachable=0    failed=0    skipped=0    rescued=0    ignored=1   

AWS ACTIONS: ['cloudcontrolapi:CreateResource', 'cloudcontrolapi:DeleteResource', 'cloudcontrolapi:GetResource', 'cloudcontrolapi:GetResourceRequestStatus', 'cloudcontrolapi:ListResourceRequests', 'rds:CreateDBCluster', 'rds:CreateDBInstance', 'rds:DeleteDBCluster', 'rds:DeleteDBInstance', 'rds:DescribeDBClusters', 'rds:DescribeDBInstances', 'rds:ListTagsForResource', 'rds:ModifyDBCluster', 'rds:RemoveFromGlobalCluster']

```